A.02.P01/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using static System.Console;
+
+internal class Program {
+   //Allows the user to predict the system generated number between 1 and 100.
+   static void Main () {
+      int n = new Random ().Next (1, 100);
+      bool isValidInput;
+      string[] guesses = { "1st", "2nd", "3rd", "4th", "5th", "6th", "last" },
+               responses = { "too low", "correct..!", "too high" };
+      WriteLine ("The number is chosen between 1 and 100. Find the number in 7 guesses!");
+      for (int i = 0, count = 7, guess; i < count; i++) {
+         var prompt = $"your {guesses[i]} guess: ";

```

Comment:
```suggestion
         var prompt = $"Your {guesses[i]} guess: ";

```
Should start with Uppercase